modules/rte_mis_allocation/src/Access/AllocationDetailEditAccessCheck.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Drupal\rte_mis_allocation\Access;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Routing\Access\AccessInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\eck\EckEntityInterface;
+use Drupal\user\UserInterface;
+
+/**
+ * Determines edit access to the school details mini node.
```

Comment:
Description doesn't match the functionality.

Reply:
modified